core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -734,7 +734,7 @@ public Map<String, Boolean> updateConfiguration(Map<String, String> propertiesMa
           Object value = key.parseValue(entry.getValue());
           Configuration.set(key, value, Source.RUNTIME);
           result.put(entry.getKey(), true);
-          successCount++;
+          changedProperties.put(key, Configuration.get(key));
```

Comment:
```suggestion
          changedProperties.put(key, value);
```

Reply:
@qian0817 There are some logic inside the `Configuration.set`, so the actually value could be changed to another one, so I get the value again to make sure it is the final value.